tests/infrared/17.1/README.md

```diff
@@ -2,16 +2,64 @@
 
 ## Basic deployment
 
-```bash
-OCP_ROUTE_IP="10.0.100.50" \
-CA_CERT_FILE_CONTENT="$(oc get secret/default-interconnect-selfsigned -o jsonpath='{.data.ca\.crt}' | base64 -d)" \
-AMQP_HOST="$(oc get route default-interconnect-5671 -ojsonpath='{.spec.host}')" \
-AMQP_PASS="$(oc get secret default-interconnect-users -o json | jq -r .data.guest | base64 -d)" \
-ENABLE_STF_CONNECTORS=true \
-ENABLE_GNOCCHI_CONNECTORS=false \
-CONTROLLER_MEMORY="24000" \
-COMPUTE_CPU="6" \
-COMPUTE_MEMORY="24000" \
-LIBVIRT_DISKPOOL="/home/libvirt/images" \
-./infrared-openstack.sh
-```
+A basic deployment can be deployed and connected to an existing STF deployment automatically after logging into the OpenShift cluster hosting STF from the host system.
+
+### Prequisites
+
+* Logged into the host system where you'll deploy the virtualized OpenStack infrastructure.
+* Installed infrared and cloned the infrawatch/service-telemetry-operator repository.
+* Updated the host systems `/etc/hosts` with the contents provided during deployment with stf-verify-containers, providing Route (DNS) resolution to the overcloud.
```

Comment:
stf-v-c is not a publicly available, so this seems a bit out of place. 

"providing Route (DNS) resolution to the overcloud" is tripping me up, I'm still not sure I'm parsing it correctly. Specifically, "overcloud" doesn't make sense to me; this is (probably?) about the host system accessing the OCP API endpoint (/etc/hosts on the host system), whereas the overcloud needs to access the STF QDR route (/etc/hosts on the overcloud nodes), and that is taken care of by the script)

I think the thrust of what you're trying to say here is...
```suggestion
* DNS resolution (or `/etc/hosts` entry) of the OpenShift cluster API endpoint
```